internal/authorization/authorization.go

```diff
@@ -32,6 +37,13 @@ func (a *Authorizer) Check(ctx context.Context, user string, relation string, ob
 	return a.client.Check(ctx, user, relation, object, contextualTuples...)
 }
 
+func (a *Authorizer) WriteTuple(ctx context.Context, user string, relation string, object string) error {
+	ctx, span := a.tracer.Start(ctx, "authorization.Authorizer.WriteTuple")
+	defer span.End()
+
+	return a.client.WriteTuple(ctx, user, relation, object)
```

Comment:
I d use the WriteTuples on the low level object of OpenFGA client, just in case we need more tuples.to be set

at the same time this is kinda blocking and should not be imho, so needs to be offloaded to a background worker/object